examples/systemcalls/systemcalls.c

```diff
@@ -93,7 +146,53 @@ bool do_exec_redirect(const char *outputfile, int count, ...)
  *   
 */
 
-    va_end(args);
+    int status;
+    pid_t pid;
     
-    return true;
+    int fd = open(outputfile, O_WRONLY|O_CREAT, 0644);
```

Comment:
@DhruvHMehta think about how you could combine logic with the function above.

Reply:
Something like this?

```
if(((fd = open(outputfile, O_WRONLY|O_CREAT, 0644)) == -1)
    return false;
```

I actually don't prefer this style because it's hard to get the parenthesis right and takes more time.
Is there any benefit to writing it this way?

Reply:
No I mean the rest of the logic in this function which is common with the function above.  I'll discuss tomorrow in class.